plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueMetastoreModule.java

```diff
@@ -175,4 +168,33 @@ else if (config.getPinGlueClientToCurrentRegion()) {
 
         return glue.build();
     }
+
+    private static Optional<StaticCredentialsProvider> getStaticCredentialsProvider(GlueHiveMetastoreConfig config)
+    {
+        if (config.getAwsAccessKey().isPresent() && config.getAwsSecretKey().isPresent()) {
+            return Optional.of(StaticCredentialsProvider.create(
+                    AwsBasicCredentials.create(config.getAwsAccessKey().get(), config.getAwsSecretKey().get())));
+        }
+        return Optional.empty();
+    }
+
+    private static StsClient getStsClient(GlueHiveMetastoreConfig config,
+                                          Optional<StaticCredentialsProvider> staticCredentialsProvider)
```

Comment:
The indentation is wrong. Please unwrap.

Reply:
Sure, should be fixed now.